fe/lib/components/LocationSuggest/components/MapSelect.tsx

```diff
@@ -0,0 +1,176 @@
+import { useQuery } from '@apollo/client';
+import {
+  Alert,
+  Box,
+  ButtonIcon,
+  Column,
+  Columns,
+  IconClear,
+  Stack,
+  Text,
+  useColor,
+  useResponsiveValue,
+} from 'braid-design-system';
+import { Map, Marker, Overlay, ZoomControl } from 'pigeon-maps';
+import React from 'react';
+import { useState } from 'react';
+import { InlineCode } from 'scoobie';
+import { useDebounce } from 'use-debounce';
+
+import { Location } from '../../../types/seekApi.graphql';
+import { formatPoint } from '../../../utils/formatPoint';
+
+import { NEAREST_LOCATIONS } from './queries';
+
+import * as styles from './styles.css';
+
+const DEFAULT_ZOOM = 10;
+
+interface Props {
+  debounceDelay?: number;
+  schemeId: string;
+  onLocationSelected: (location: Location) => void;
+  initialLocation: [number, number];
+}
+
+export const MapSelect = ({
+  debounceDelay = 250,
+  schemeId,
+  onLocationSelected,
+  initialLocation,
+}: Props) => {
+  const [center, setCenter] = useState(initialLocation);
+  const [latLong, setLatLong] = useState(initialLocation);
+  const [showSuggestions, setShowSuggestions] = useState(false);
+
+  const responsivevalue = useResponsiveValue();
+  const color = useColor();
+  const [debouncedLatLong] = useDebounce(latLong, debounceDelay);
+
+  const { data, previousData } = useQuery<{
+    nearestLocations: Array<Location> | null;
+  }>(NEAREST_LOCATIONS, {
+    variables: {
+      geoLocation: {
+        latitude: debouncedLatLong[0],
+        longitude: debouncedLatLong[1],
+      },
+      first: 3,
+      schemeId,
+    },
+    fetchPolicy: 'no-cache',
```

Comment:
Is this still necessary with the new design?

Reply:
Nope, will remove 👍